packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/ProjectController.java

```diff
@@ -1120,6 +1130,9 @@ public ResponseEntity<JsonNode> updateProjectUserPermissions(
 
 			final RebacProject what = new RebacProject(projectId, reBACService);
 			final RebacUser who = new RebacUser(userId, reBACService);
+
+			// no need to update the search perms since we cant never reduce below a read.
```

Comment:
double negative

Reply:
I've never not heard something that doesn't say that this isn't what you don't mean, or isn't it?